Admin/AdminBuilder.php

```diff
@@ -22,7 +22,7 @@
 /**
  * @author Christian Gripp <mail@core23.de>
  */
-final class AdminBuilder implements FormBuilderInterface, DatagridBuilderInterface
+final class AdminBuilder implements AdminBuilderInterface
```

Comment:
Why would you mock this class? You should mock `FormBuilderInterface` or `DatagridBuilderInterface` if you need them.